debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/view/dtfj/java/DTFJJavaRuntime.java

```diff
@@ -929,13 +930,27 @@ public String getFullVersion() throws CorruptDataException {
 	}
 
 	public String getVersion() throws CorruptDataException {
+		String version = null;
+		J9RASPointer ras = null;
 		try {
+			ras = DTFJContext.getVm().j9ras();
 			//starting with 26 stream, RAS structure contains the JRE version which avoids the potential pitfall
 			//of using system properties which may have their name and/or value stored in native libraries
-			return DTFJContext.getVm().j9ras().serviceLevel().getCStringAtOffset(0);
+			version = DTFJContext.getVm().j9ras().serviceLevel().getCStringAtOffset(0);
 		} catch (Throwable t) {
 			throw J9DDRDTFJUtils.handleAsCorruptDataException(DTFJContext.getProcess(), t);
 		}
+		try {
+			U8Pointer productName = ras.productName();
+			if (productName.notNull()) {
+				version = version + " " + productName.getCStringAtOffset(0);
+			}
+		} catch (NoSuchFieldException e) {
+			// ignore if the product name doesn't exist
+		} catch (com.ibm.j9ddr.CorruptDataException e) {
+			throw J9DDRDTFJUtils.handleAsCorruptDataException(DTFJContext.getProcess(), e);
+		}
+		return version;
```

Comment:
The point of creating `ras` was to avoid repeated calls to `DTFJContext.getVm().j9ras()`.
There's no need to initialize `version = null`.
There's no need to catch `com.ibm.j9ddr.CorruptDataException` if the version is fetched in a nested `try` statement.
In the end, this is what I think we want:
```java
		try {
			//starting with 26 stream, RAS structure contains the JRE version which avoids the potential pitfall
			//of using system properties which may have their name and/or value stored in native libraries
			J9RASPointer ras = DTFJContext.getVm().j9ras();
			String version = ras.serviceLevel().getCStringAtOffset(0);
			try {
				U8Pointer productName = ras.productName();
				if (productName.notNull()) {
					version = version + " " + productName.getCStringAtOffset(0);
				}
			} catch (NoSuchFieldException e) {
				// ignore if the product name doesn't exist
			}
			return version;
		} catch (Throwable t) {
			throw J9DDRDTFJUtils.handleAsCorruptDataException(DTFJContext.getProcess(), t);
		}
```

Reply:
Updated it, and the import as well.